default-templates/new-account/.acl

```diff
@@ -1,23 +1,25 @@
 # Root ACL resource for the user account
 @prefix acl: <http://www.w3.org/ns/auth/acl#>.
 
-<#owner>
+# The homepage is readable by the public
+<#public>
     a acl:Authorization;
+    acl:agentClass foaf:Agent;
+    acl:accessTo </>;
```

Comment:
Should we state that `index.html` is publicly readable as well, since we're deleting `index.html.acl`? 

If we don't, people won't be able to access `/index.html`, but they would be able to access `/`. That might be a use case that's not that important anyway though, so might not be necessary.

Reply:
Hmm, was planning to say we could change line 8 to `acl:accessTo </>, </index.html>`, but that doesn't work either...

Reply:
I would refrain from including anything `/index/` in the ACL files. As far as the user is concerned, these don't really (need to) exist. They are a convention; we could start naming them _default.html_ in the future.

Reply:
It's part of the broader point in https://github.com/solid/web-access-control-spec/issues/36.